library/src/scala/quoted/Quotes.scala

```diff
@@ -33,24 +33,27 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
     */
     def matches(that: Expr[Any]): Boolean
 
-    /** Return the unlifted value of this expression.
+    /** Return the value of this expression.
      *
      *  Returns `None` if the expression does not contain a value or contains side effects.
```

Comment:
```suggestion
     *  Returns `None` if the expression does not evaluate to a value or possibly contains side effects.
```

Reply:
The evaluation word should be kept out of the docs as this leads to confusion